lang/en/cms.php

```diff
@@ -37,7 +37,13 @@
 $string['maxgrade'] = 'Default max grade';
 $string['maxgrade_desc'] = 'The default max grade when creating a new custom content type instance.';
 $string['modulename'] = 'CMS';
+$string['modulenameplural'] = 'CMS';
 $string['newcontenttype'] = 'Add new content type';
 $string['pluginname'] = 'CMS';
+$string['privacy:metadata:cms'] = 'Custom content type instances';
+$string['privacy:metadata:cms:usermodified'] = 'User who modified the instances';
+$string['privacy:metadata:cms_types'] = 'Custom content types';
+$string['privacy:metadata:cms_types:usermodified'] = 'User who modified the custom content types';
 $string['settings'] = 'Custom content type settings';
 $string['table:name'] = 'Custom content type';
+$string['cms:addinstance'] = 'Add a new instance';
```

Comment:
Might want to be a bit more explicit here that we're adding a new CMS instance or Custom content type instance